### PR TITLE
umm I fixedDatSearch so like if you search large youll see

### PR DESCRIPTION
**Pull request by NoahBazaar**
hey i want this code to be in main the main branch and its ready and its done... its final it should go in main branch

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -210,9 +210,14 @@ function search(searchString, section = 'all') {
         items.forEach(item => {
             const itemName = item.getAttribute('data-name') || '';
             const itemText = item.textContent || '';
```
Comment (oceanseth) [inline script.js:212]: this should be just item.text. And we should also search item.bottomtext

Comment (oceanseth) [reply script.js:212]: const itemText = item.textContent + " " + item.bottomtext

Comment (NoahBazaar) [reply script.js]: DONE
YERP

**Review by oceanseth — APPROVED** [review]
lgtm!

**Pull request merged**